OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ChildController.cs

```diff
@@ -235,7 +235,7 @@ public async Task<IActionResult> CreateChildren(List<ChildDto> childrenDtos)
     [ProducesResponseType(StatusCodes.Status403Forbidden)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     [HttpPut]
-    public async Task<IActionResult> Update(ChildDto dto)
+    public async Task<IActionResult> Update(ChildUpdateDto dto)
```

Comment:
probably need to add Id as path parameter. Also probably the good time to make this more REST :) like (i will share excel file to Discord with more details)

get | /api/v1/Child/GetAllForAdmin | change | get | /api/v1/children
-- | -- | -- | -- | --
get | /api/v1/Child/GetChildrenListByParentId/{parentId}/{isParent} | change | get | /api/parents/{parentId}/children?{isParent=true/false}
get | /api/v1/Child/GetByParentIdForAdmin/{parentId} | change | get | TODO
get | /api/v1/Child/GetUsersChildren | change | get | /api/v1/children/my
get | /api/v1/Child/GetUsersChildById/{id} | change | get | /api/v1/children/my/{id}
get | /api/v1/Child/GetApprovedByWorkshopId/{workshopId} | change | get | TODO
post | /api/v1/Child/Create | change | post | /api/v1/children
put | /api/v1/Child/Update | change | put | /api/v1/children/{id}
delete | /api/v1/Child/Delete/{id} | change | delete | /api/v1/children/{id}

Reply:
done